src/agent/filecontainerhelper.ts

```diff
@@ -13,6 +14,29 @@ export function copyToFileContainer(taskCtx: ctxm.TaskContext, localPath: string
 	return fc.copyToFileContainer(localPath, containerId, containerFolder);
 }
 
+export function getUploadHeaders(isGzipped: boolean, 
```

Comment:
kind of a hack. Ideally the generator would be able to determine all of the headers that are required for every operation, but I couldn't figure out how to programmatically determine which operations require encoding, contentId, and length headers. Encoding could probably be done with an attribute on controllers but others...